integration/messaging/test/utils/sendMessage.js

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-const fetch = require('node-fetch');
+import { fetch } from 'node-fetch';
```

Comment:
`node-fetch` is now an ESM-only module - you are not able to import it with require. See node-fetch is an ESM-only module - you are not able to import it with require.